spec/spec_helper.rb

```diff
@@ -0,0 +1,96 @@
+# This file was generated by the `rails generate rspec:install` command. Conventionally, all
+# specs live under a `spec` directory, which RSpec adds to the `$LOAD_PATH`.
+# The generated `.rspec` file contains `--require spec_helper` which will cause
+# this file to always be loaded, without a need to explicitly require it in any
+# files.
+#
+# Given that it is always loaded, you are encouraged to keep this file as
+# light-weight as possible. Requiring heavyweight dependencies from this file
+# will add to the boot time of your test suite on EVERY test run, even for an
+# individual file that may not need all of that loaded. Instead, consider making
+# a separate helper file that requires the additional dependencies and performs
+# the additional setup, and require it from the spec files that actually need
+# it.
+#
+# See https://rubydoc.info/gems/rspec-core/RSpec/Core/Configuration
+RSpec.configure do |config|
```

Comment:
- [ ] A number of tests are not working for me:

![image](https://user-images.githubusercontent.com/57421823/194123592-92245b12-e219-4851-88f0-e131c7e4f59e.png)

Please, consider debugging those tests, and check if they work on your environment.  

That way, your app's integrity and health over time will be assured. 💪

Reply:
- [x] Done, thanks for pointing this out 👍